e2e-tests/gef/cypress/e2e/pages/bank-review-date.js

```diff
@@ -9,6 +11,16 @@ const bankReviewDate = {
   bankReviewDateDetails: () => cy.get('[data-cy="bank-review-date-details"]'),
   continueButton: () => cy.get('[data-cy="continue-button"]'),
   saveAndReturnButton: () => cy.get('[data-cy="save-and-return-button"]'),
+
+  /**
+   * Fill in the bank review date form
+   * @param {Date} date
+   */
+  fillInBankReviewDate: (date) => {
+    this.bankReviewDateDay().clear().type(format(date, 'd'));
+    this.bankReviewDateMonth().clear().type(format(date, 'M'));
+    this.bankReviewDateYear().clear().type(format(date, 'yyyy'));
+  },
```

Comment:
Is this pattern (putting functions into `page` selectors), something we have in other places?

I'm not a fan of this because:

1) Page selectors are no longer pure selectors
2) Individual tests have to import the page in order to call the function
3) Makes it difficult to move to another place in the future

Unless this is a pattern in other places, could we move this to a non-page specific location, i.e a standalone cypress command? So it can be called like this for example:

```js
cy.completeBankReviewDate();
```

"fillIn" or "complete" or something else is up for debate I think.